contrib/localnet/scripts/start-zetaclientd.sh

```
@@ -69,7 +69,10 @@ RELAYER_KEY_PATH="$HOME/.zetacored/relayer-keys"
 mkdir -p "${RELAYER_KEY_PATH}"
 
 mkdir -p "$HOME/.tss/"
-zetae2e local get-zetaclient-bootstrap > "$HOME/.tss/address_book.seed"
+address_book_path="$HOME/.tss/address_book.seed"
+if [[ ! -f $addresss_book_path ]]; then
```

Shellcheck (0.10.0)

[warning] addresss_book_path is referenced but not assigned (did you mean 'address_book_path'?).

(SC2154)